src/extension/inspire/src/main/java/org/geoserver/inspire/web/UniqueResourceIdentifiersEditor.java

```diff
@@ -152,24 +152,6 @@ protected void onError(AjaxRequestTarget target, Form<?> form) {
         };
         add(button);
         
-        // grab a seat... the way I'm adding this validator in onBeforeRender will be hard
-        // to stomach... however, could not find other way to add a validation to an editabl table, grrr
-        add(new IValidator<UniqueResourceIdentifiers>() {
-
-            @Override
-            public void validate(IValidatable<UniqueResourceIdentifiers> validatable) {
-                UniqueResourceIdentifiers identifiers = identifiersModel.getObject();
-                if(identifiers.size() == 0) {
-                  ValidationError error = new ValidationError();
-                  String message = new ParamResourceModel(
-                          "noSpatialDatasetIdentifiers", UniqueResourceIdentifiersEditor.this).getString();
-                  error.setMessage(message);
-                  validatable.error(error);
-                }
-                
-            }
-            
-        });
```

Comment:
I guess the nicest thing would be to have one of those yellow warning triangles with a text message saying that no element will be produced when insufficient parameters have been supplied similar to the warnings about not having set a strong password etc. I'm not familiar enough with the wicket stuff to know how to do that immediately although I'm sure I could have a look at some of the other examples to get a hint. I'll have a look today.